test/TestProjects/Parameters-LowLevel/src/Generated/Docs/ParametersLowlevelClient.xml

```diff
@@ -8,17 +8,7 @@ This sample shows how to call NoRequestBodyResponseBodyAsync with required param
 var credential = new AzureKeyCredential("<key>");
 var client = new ParametersLowlevelClient(credential);
 
-Response response = await client.NoRequestBodyResponseBodyAsync(1234, 1234);
-
-JsonElement result = JsonDocument.Parse(response.ContentStream).RootElement;
-Console.WriteLine(result.ToString());
-]]></code>
-This sample shows how to call NoRequestBodyResponseBodyAsync with all parameters, and how to parse the result.
-<code><![CDATA[
-var credential = new AzureKeyCredential("<key>");
-var client = new ParametersLowlevelClient(credential);
```

Comment:
Why your change would impact the docs?

Reply:
We will generate two sample calls for each method. One only has required parameters and the other has required parameters and optional parameters. Since we change all the parameters to required, then only one sample call left.